core/src/main/java/com/gdx/game/screen/OptionScreen.java

```diff
@@ -59,33 +82,234 @@ private void loadContents() {
             vfxManager.addEffect(vfxEffect);
         }
 
+        GameScreen.GameState gameState = GameScreen.getGameState();
+
         optionTable = createTable();
-        handleControlButton();
+        if (gameState == null) handleControlButton();
```

Comment:
i want to keep the possibility that the player can update his settings while playing, so no if condition :)